src/TodoList.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import TodoItem from "./TodoItem";
+import "./todo-list.css";
+
+const TodoList = (props) => {
+  return (
+    <>
+      <div className="list-container">
+        {props.isCompletedShown
+          ? props.data.map((item) => {
+              return (
+                <>
+                  <TodoItem
+                    key={item.id}
+                    name={item.itemName}
+                    id={item.id}
+                    status={item.itemStatus}
+                    onItemChange={props.onItemChange}
+                  />
+                </>
+              );
+            })
+          : props.data.map((item) => {
```

Comment:
Would be better to not pass in isCompletedShown, but just have App filter the items based on whether completed are shown or not.  That is, it's up to the app to decide what items to show (since it's in charge of the button).